Gopkg.lock

```diff
@@ -7,6 +7,12 @@
   revision = "0dd823580c78a79ae9696eb9b3650e400fff140f"
 
 [[projects]]
+  name = "github.com/Microsoft/go-winio"
```

Comment:
Was this pulled in as a recursive dep from `osquery-go`?

Reply:
Yeah, I believe the `Gopkg.lock` also declares transitive deps as well, but notice it's not in `Gopkg.toml`